README.md

```diff
@@ -0,0 +1,98 @@
+[![uv](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/uv/main/assets/badge/v0.json)](https://github.com/astral-sh/uv)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+[![python](https://img.shields.io/badge/Python-3.12-3776AB.svg?style=flat&logo=python&logoColor=yellow)](https://www.python.org)
+[![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
+[![formula1-race-analysis-build](https://github.com/KrisKulbych/formula1-race-analysis/actions/workflows/build.yaml/badge.svg)](https://github.com/KrisKulbych/formula1-race-analysis/actions/workflows/build.yaml)
+
+# Formula 1 Race Analysis
+Formula 1 Race Analysis is a command-line tool for analyzing and displaying the results of Formula 1 qualifying sessions, specifically Q1. It provides sorting, filtering, and reporting options to analyze driver performance effectively.
+
+****This project is at the early stages of development. 
+
+## Installation
+Copy and run the following command:
+```console
+pip install -i https://test.pypi.org/simple/ formula1-race-analysis
+```
+Formula 1 Race Analysis supports Python >= 3.12.
+
+## Currently supported tools:
+- pre-commit;
+- ruff;
+- uv;
+- pytest.
+
+## Interface
+The primary command to use is generate_report. It takes the following options:
+```console
+generate_report --data_dir <PATH_TO_DATA> [--order asc|des] [--driver DRIVER_NAME]
```

Comment:
![image](https://github.com/user-attachments/assets/d9c09ac2-94e9-41ba-85d4-7ca980bb8e70)
Apparently it is no the correct command